### PR TITLE
Add Dygraph to static demo

### PR DESCRIPTION
**Pull request by Yelrose**

### CODE REVIEW

File: examples/deploy_gnn/convert_to_static.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    dtype="float32")
+
+output = model(num_nodes, edges, feature)
+
+place = paddle.CPUPlace()
+exe = static.Executor(place)
+exe.run(static.default_startup_program())
+
+# Load DyGraph Model
+state_dict = paddle.load("gnn.pdparam")
```
Comment (ZHUI) [inline examples/deploy_gnn/convert_to_static.py:62]: 这里的参数 dict ，里面的key不需要转一遍吗？

File: examples/deploy_gnn/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+```
+python train.py
+```
+
+
+# Write GNN in convert_to_static and load state_dict
+
+```
+python convert_to_static.py
```
Comment (ZHUI) [inline examples/deploy_gnn/README.md:14]: 示例 `convert_to_static.py` 是跑静态图代码。部署的话，目前推荐用法是 jit.save 导出静态图模型，进行部署，看这个是否咱们PGL需要加上？

File: tests/test_static_graph.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    def forward(self, num_nodes, edges, feature):
+        graph = pgl.Graph(num_nodes=num_nodes, edges=edges)
+        for fn in self.conv_fn:
+            feature = fn(graph, feature)
+        output = self.pool_fn(graph, feature)
+        return output
+
+
+class StaticGraphOpTest(unittest.TestCase):
```
Comment (ZHUI) [inline tests/test_static_graph.py:43]: 这个测试是为了保证，pgl的Graph对静态图有用是吗？目前咱们Graph的method很多，是否很多method在静态图的支持上会有风险呢？

File: examples/deploy_gnn/convert_to_static.py (hunk 1 above)
Comment (wawltor) [reply examples/deploy_gnn/convert_to_static.py:62]: 用一下 jit.save来转一下试试看看吧  可以参考这里 https://github.com/PaddlePaddle/PaddleNLP/blob/develop/examples/language_model/bert/export_model.py

File: examples/deploy_gnn/convert_to_static.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+import pgl
+import numpy as np
```
Comment (wawltor) [inline examples/deploy_gnn/convert_to_static.py:17]: numpy按照规范提前吧

File: tests/test_static_graph.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        nfeat = np.random.randn(num_nodes, dim)
+
+        feed_dict = {
+            "num_nodes": num_nodes,
+            "edges": np.array(
+                edges, dtype="int32"),
+            "feature": nfeat.astype("float32"),
+        }
+        out = exe.run(prog, feed=feed_dict, fetch_list=[output])
+
```
Comment (wawltor) [inline tests/test_static_graph.py:70]: 这里好像没有保证正确性，单测能不能比对一下动态图，静态图的forward的输出结果是否一致了，这样才能保证部署结果的正确性

**Review by ZHUI — APPROVED** [review]
LGTM
